C-Plus-Plus/other/max_possible_subsequence.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+Given two words and a string.
+By using two words , Count How many different subsequence's can be created through the string.
+Two subsequence's are considered to be different if they start or end
+at different positions in the string.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+

```

Comment:
Add one line description about function above it.

Reply:
i have already written it :/